ci-operator/step-registry/openshift/e2e/aws/ccm/openshift-e2e-aws-ccm-workflow.yaml

```diff
@@ -2,18 +2,31 @@ workflow:
   as: openshift-e2e-aws-ccm
   steps:
     pre:
-    - chain: ipi-aws-pre
+    - ref: ipi-conf
+    - ref: ipi-conf-aws
+    # TODO: add - ref: ipi-install-monitoringpvc
+    # We do not include monitoring PVC introduced in https://github.com/openshift/release/pull/15040
+    # as functionality of this PVC depends on successfull migration to CSI we are currently unable to
+    # fully determine. 
+    - chain: ipi-install
     - ref: ccm-conf-apply-feature-gate
+    - ref: storage-conf-csi-aws-ebs
+    - ref: storage-conf-wait-for-csi-migration
     test:
     - ref: openshift-e2e-test
     post:
     - chain: gather-core-dump
     - chain: ipi-aws-post
     env:
+      TEST_INSTALL_CSI_DRIVERS: aws-ebs
+      TEST_CSI_DRIVER_MANIFEST: manifest-aws-ebs.yaml
+      MCO_IMAGE_OVERRIDE: "quay.io/dgrigore/machine-config-operator:ccm"
+      KCMO_IMAGE_OVERRIDE: "quay.io/dgrigore/cluster-kube-controller-manager-operator:ccm"
```

Comment:
I wonder if it would be preferable to use actual SHAs here, otherwise a push to update these images could break the testing, possibly better to submit PRs to update them as and when we think there are significant changes?

Reply:
+1, using SHAs will help keep the images in order on the test cluster

Reply:
Fixed.